bindings/rust/integration/benches/s2nc.rs

```diff
@@ -2,20 +2,21 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use criterion::{criterion_group, criterion_main, Criterion};
-use std::{dbg, env, process::Command, time::Duration};
-mod utils;
+use std::{env, process::Command, time::Duration};
 
 pub fn s2nc(c: &mut Criterion) {
     let mut group = c.benchmark_group("s2nc");
+    /*
+    example S2NC_ARGS:
+    "--non-blocking -T -f ../integration/trust-store/ca-bundle.crt -c test_all_tls13 www.netflix.com 443"
+    Set in tests/integrationv2/providers.py line 379-ish
+    */
```

Comment:
The line number will definitely change :) "Set in the S2N provider" or "Set in the S2N provider in tests/integrationv2/providers.py" is probably sufficient.

But this is really brittle. Won't it break if we add a new argument to the s2n call in the integration tests? Or remove an argument? Like "--non-blocking" is a relatively recent addition, and as a developer I wouldn't expect moving "-c test_all_tls13" to the beginning of the command to break anything. And this definitely won't work for any tests other than well_known_endpoints. 

If you really do need to do this, you'd also need a comment in the S2N provider warning developers about modifying the arguments without updating this script.

Reply:
Fixed the comment.  From @WesleyRosenblum's suggeston- moving the test name parsing into the provider addresses the fragility issue and keeps these handlers from caring about order or position.